lib/release_notes_bot/persists.ex

```diff
@@ -221,7 +221,15 @@ defmodule ReleaseNotesBot.Persists do
 
   defp drop_question_mark(string), do: String.replace(string, "?", "")
 
-  defp update_confluence_page() do
+  defp get_headers(token) do
+    [
+      {"Content-Type", "application/json"},
+      {"Authorization", "Basic #{token}"}
```

Comment:
For functions like these, that setup data structures based off input, we should provide a data structure _as_ input so it's not only more flexible in the future but it's clear to the caller what we're passing.
Instead we could do
```ex
defp get_headers(%{"token" => token}) do
#
end

# more explicit caller
get_headers(%{"token" => data.token})

# future iterations don't have to worry about argument order
get_headers(%{"cache_control" => "no-cache", "token" => token})
```

Reply:
This is sexy - I'm definitely taking a page from this :)